README.md

```diff
@@ -4,13 +4,37 @@ This repository contains all the operational and user documentation for running
 
 ## Getting Started
 
-We have the [Open Data Hub](https://opendatahub.io/) applications deployed and running in a MOC ([Mass Open Cloud](https://massopen.cloud/)) cluster. All user documentation such as the login process for each of the applications deployed can be found [here](https://github.com/operate-first/odh-moc-support/tree/main/docs/user-docs).
+We have [Open Data Hub](https://opendatahub.io/) applications deployed and running in a MOC ([Mass Open Cloud](https://massopen.cloud/)) cluster.
+
+The ODH homepage on MOC can be found at [ODH Dashboard](https://odh-dashboard-opf-dashboard.apps.cnv.massopen.cloud/). From there you can access any of the provided application:
+
+https://odh-dashboard-opf-dashboard.apps.cnv.massopen.cloud/
+
+![ODh Dashboard](docs/assets/images/odh-dashboard.png)
+
+## Component status
+
+| Component               | Availability      | Authentication                                                                     |
+| ----------------------- | ----------------- | ---------------------------------------------------------------------------------- |
+| AI-Library              | ❌ (Not deployed) |                                                                                    |
+| Airflow                 | ❌ (Not deployed) |                                                                                    |
+| Argo                    | ✔️                | None (Login via Openshift [WIP](https://github.com/operate-first/odh/issues/39))   |
+| Grafana                 | ✔️                | Custom (Login via Openshift [WIP](https://github.com/operate-first/odh/issues/37)) |
+| Hue                     | ❌ (Stabilizing)  | Custom (Login via Openshift [WIP](https://github.com/operate-first/odh/issues/47)) |
+| JupyterHub              | ✔️                | Login via Openshift                                                                |
```

Comment:
should we maybe rephrase it as `Login via OpenShift using your Gmail account`?

Reply:
Google account, perhaps?

Reply:
I think you can use any account provider mapped to Openshift Auth, can't you? Like.. if you have UNI login, you can use that, or not?

Reply:
I'm not actually sure... I thought most of our rbac was tied to things like "<person>@redhat.com" and all that.

Reply:
yes.. we do.. but we probably don't expect students/uni researchers to do that.

Reply:
yeah, so just to avoid confusion between users thinking that they will need to have a separate account, I was thinking we could mention it as something like: `Login via OpenShift -> moc-sso -> your email account`? we could also include screenshots, if that makes it easier

Reply:
Ultimately, the link they're clicking at each of the service says "Login via Openshift"

![image](https://user-images.githubusercontent.com/7453394/101056064-dfb6f300-358a-11eb-8d0a-c7598be06a39.png)
![image](https://user-images.githubusercontent.com/7453394/101056116-ee9da580-358a-11eb-9f0e-0ee451c954de.png)

Which leads to the Openshift provider selection - we probably should mention the `moc-sso` here.
![image](https://user-images.githubusercontent.com/7453394/101057100-f742ab80-358b-11eb-9765-81b0cb00aa65.png)


The path it takes from there is really out of our control. If they use UNI login or email account, that's transparent to us and we really shouldn't care. We can suggest them to select the appropriate login "type" but I think saying "if you don't have an UNI account, click the other button and use Google account " is not very helpful. :smile: It needs better wording. :smile:

Reply:
@hema, @anishasthana how do you like this?

Reply:
Yeah I think that makes sense.

Reply:
yes I think this looks much better! :+1: :smile: